testing/minikube/jupyterhub-helm-config.yaml

```diff
@@ -3,80 +3,23 @@
 
 # the binder apiToken bit requires jupyterhub helm-chart v0.5.x
 
-rbac:
-  enabled: true
+custom:
+  cors:
+    allowOrigin: "*"
+
 hub:
   cookieSecret: "36091e950f98f033aeae2520e2fe4c8599bc391598f910e510ad670765fbc1ff"
   db:
     type: "sqlite-memory"
   services:
     binder:
-      admin: true
```

Comment:
Isn't this what makes the binder service an "admin" in the eyes of JupyterHub?

Reply:
I think, with the changes in `install-hub` file (https://github.com/jupyterhub/binderhub/pull/1023/files#diff-971e2aef3f7ade4d8bed25e7e6d1c858) now it is taken from values.yaml. So this line can be deleted

Reply:
That's right, it's merged with the default config so only the overrides are needed.